tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -125,34 +128,41 @@ def mirror_daemon(self, enable=None, start=None, stop=None, restart=None):
 
     # Initial setup of mirroring host
     def setup_mirror(self, peer_cluster, **kw):
-        self.exec_cmd(ceph_args=False, cmd="yum install -y rbd-mirror")
+        if self.ceph_version >= 5:
+            self.ceph_client.exec_command(
+                sudo=True,
+                cmd=f"ceph orch apply rbd-mirror --placement={self.ceph_rbd_mirror.hostname}",
```

Comment:
Can we add `rbd-mirror` to `CephAdm` class? We could reuse the existing workflow